plugins/inputs/snmp/netsnmp.go

```diff
@@ -0,0 +1,256 @@
+package snmp
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"log" //nolint:revive
+	"os/exec"
+	"strings"
+	"sync"
+
+	"github.com/influxdata/wlog"
+)
+
+//struct that implements the translator interface. This calls existing
+//code to exec netsnmp's snmptranslate program
+type netsnmpTranslator struct {
+}
+
+func NewNetsnmpTranslator() *netsnmpTranslator {
+	return &netsnmpTranslator{}
+}
```

Comment:
Could maybe return error when net-snmp commands are not found?

Reply:
Could be a good idea, but this isn't how it worked before so I'm not going to do it in this PR. I'd hate to break things for people who (for example) only use numeric oids.

Reply:
You know telegraf is doing lookups in that case as well, right? (in order to get a name and conversion)